bin/install.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+appendIfNotExist() {
+  if ! grep -Fxq "$1" $HOME/.zshrc; then
+    if $2; then
+      echo "Appending $1 to .zshrc"
```

Comment:
should we use `tee` to write to a log file as well? 

Also I think the false param is only used for Start and End headers, if we plan on writing that to stdout too, we can use tee for all three

http://man7.org/linux/man-pages/man1/tee.1.html
https://linux.101hacks.com/unix/tee-command-examples/

I would suggest log for everything that is being echoed to stdout so that we have logs for future debug.

Reply:
I will create a new issue for unified logging.